addons/core/XEH_preInit.sqf

```diff
@@ -189,9 +189,10 @@ If (isServer && {isMultiplayer || is3DENMultiplayer}) then {
     "",
     1,
     {
-        TFAR_defaultFrequencies_sr_west = [_this, TFAR_MAX_CHANNELS, TFAR_MAX_SW_FREQ, TFAR_MIN_SW_FREQ, TFAR_FREQ_ROUND_POWER] call DFUNC(parseFrequenciesInput);
-        publicVariable "TFAR_defaultFrequencies_sr_west";
-        TFAR_setting_defaultFrequencies_sr_west = str(TFAR_defaultFrequencies_sr_west apply {parseNumber _x});
+        if (isServer) then {
+            TFAR_defaultFrequencies_sr_west = [_this, TFAR_MAX_CHANNELS, TFAR_MAX_SW_FREQ, TFAR_MIN_SW_FREQ, TFAR_FREQ_ROUND_POWER] call DFUNC(parseFrequenciesInput);
+            publicVariable "TFAR_defaultFrequencies_sr_west";
```

Comment:
Currently the biggest problem with default frequencies is that publicVariables don't arrive soon enough on the client.
Instead of doing this on the server and trying to push it to the client and hoping it arrives soon enough.. Just do it on everyone including the client.

Reply:
My Problem with this solution is, that the clients could get different settings.
e.g.
The mission defines only the first 2  frequencies.
Then everyone fills the predefined frequencies with his/her own frequencies.
I consider this as not wanted behavior.

I would prefer if the clients don't do anything until the at least the "CBA_settingsInitialized" Event was raised.

Reply:
The code will only be executed when the settings are done.
And afaik it waits till it is sure that the server settings arrived if there are any.